File: app/src/main/java/com/kazumaproject/screenshothelper/service/NavigationService.java
package com.kazumaproject.screenshothelper.service;

import android.accessibilityservice.AccessibilityService;
import android.view.accessibility.AccessibilityEvent;

public class NavigationService extends AccessibilityService {
    public static NavigationService navigationService;

    public void onAccessibilityEvent(AccessibilityEvent accessibilityEvent) {
    }

    public void onInterrupt() {
    }


    public void onServiceConnected() {
        super.onServiceConnected();
        navigationService = this;
    }
}

File: app/src/main/java/com/kazumaproject/screenshothelper/service/NavigationService.kt
package com.kazumaproject.screenshothelper.service

import android.accessibilityservice.AccessibilityService
import android.view.accessibility.AccessibilityEvent

class NavigationService: AccessibilityService() {

    lateinit var navigationService : NavigationService

    fun isInitialize(): Boolean{
        return this::navigationService.isInitialized
    }

    override fun onAccessibilityEvent(event: AccessibilityEvent?) = Unit

    override fun onInterrupt() = Unit

    override fun onServiceConnected() {
        super.onServiceConnected()
        navigationService = this
    }
}